PosteAPI/models.py

```diff
@@ -28,9 +57,9 @@ def __str__(self):
 
 
 class Post(models.Model):
-    title = models.CharField(max_length=100, blank=True)
+    title = models.CharField(max_length=100, blank=False)
     description = models.TextField(blank=True)
-    url = models.CharField(max_length=1000, blank=False)
+    url = models.CharField(max_length=1000)
```

Comment:
I think `url` should also be required:
```suggestion
    url = models.CharField(max_length=1000, blank=False)
```

Reply:
As requested.